specs/grafana-integration.adoc

```diff
@@ -0,0 +1,192 @@
+*  Update tendrl-ansible/roles/tendrl-server/handlers/main.yml to check the
+   status of grafana server service and start
+
+Reference issue: Tendrl/tendrl-ansible#18
+
+
+==== Tendrl/monitoring-integration impact:
+This is the new package added into tendrl app to dynamically configure
+grafana by using the json template with the given values or the retrieved
+values from etcd. The json will be used by the grafana service which will
+fetche the values from graphite to visualize the data. There will be a
+service which runs in the tendrl-server to perform this job.
+This package needs to be builded for tendrl.
+This package (rpm) should install all the required packages and the
+service file during installation.
+Reference Doc: https://github.com/Tendrl/specifications/pull/218
+
+
+==== Tendrl/node_agent impact:
+Node-monitoring will be integrated with node-agent. The services of
+both the components will be merged together. The changes should be
+updated into the spec file, after the merge of the pr for the
+spec: https://github.com/Tendrl/specifications/pull/219.
+Build scripts should be updated to support make-snapshot which should
+automatically update the release version as per fedora guidelines to
+ntegrate with centos-ci
+
+
+==== Tendrl/api impact:
+This package will be used for calling api to importing gluster and
+user identity management. The build script should be updated to support
+make-snapshot and automatic release version.
+
+
+==== Tendrl/gluster-integration impact:
+Build scripts should be updated to support make-snapshot which should
+automatically update the release version as per fedora guidelines to
+integrate with centos-ci.
+
+
+==== Tendrl/ui impact:
+This package will be used to show tendrl dashboard which will show the
+appropriate grafana dashboard based on user’s selection.
+This tendrl component is now renamed to tendrl-ui. This change should be
+updated everywhere in copr, jenkins and in the tendrl-ui spec files.
+Add required changes for make-snapshot support and automatic version
+change as per guidelines.
+
+
+==== Notifications/Monitoring impact:
+Remove all the build releated script for performance monitoring,
+node-monitoring from copr, jenkins ci.
+
+
+==== Tendrl/common impact:
+Update the scripts for automatic version change as per guidelines.
+
+
+==== Sds integration impact:
+None
+
+=== Security impact:
+None
+
+
+=== Other end user impact:
+None
+
+
+=== Performance impact:
+None
+
+
+=== Other deployer impact:
+None
+
+
+=== Developer impact:
+None
+
+
+=== Assignee(s):
+Timothy Asir Jeyasing
+
+
+=== Work Items:
+None
+
+
+== Dependencies:
+None
+
+== Testing:
+Test whether the package installation installs all the packages properly
+
+
+== Documentation impact:
+Document should be updated to provide installation steps of grafana
+and monitoring-integration
+
+Steps
+1) Add the repo for grafana
+   ex:-
+   [grafana]
+   name=grafana
+   baseurl=https://packagecloud.io/grafana/stable/el/6/$basearch
```

Comment:
I haven't tried that yet, but this looks like repo for RHEL/CentOS 6, is it the case?